raiden-ts/src/helpers.ts

```diff
@@ -95,10 +95,11 @@ export const getContracts = (network: Network): ContractsInfo => {
  *
  * @param network - Network to include in message
  * @param main - Main signer to derive subkey from
+ * @param origin_url - URL of the origin to generate the subkey for
  * @returns Subkey's signer & address
  */
-async function genSubkey(network: Network, main: Signer) {
-  const url = globalThis?.location?.origin ?? 'unknown';
+async function genSubkey(network: Network, main: Signer, origin_url?: string) {
```

Comment:
Please, use camelCase only for variables and parameters. Snake_case is allowed only for API objects or uppercased constants

Reply:
Oh damn, yes of course! I wrote too much Python privately the last days. :see_no_evil: 
But why does linting not report? :scream:

Reply:
:heavy_check_mark: